src/pybind/mgr/requirements-mgr.txt

```diff
@@ -0,0 +1,6 @@
+CherryPy==18.8.0
+pyjwt==2.4.0
+routes==2.5.1
+werkzeug==2.0.3
+saml==0.9.0
+setuptools==59.6.0
```

Comment:
Are these the latest or the current ones?

Reply:
I've updated:
cherrypy from 18.4.0 > 18.8.0
saml 1.9.0 > 0.9.0, looks like a downgrade but pip install has these versions: `0.2.0, 0.2.1, 0.3.0, 0.3.1, 0.3.2, 0.3.3, 0.4.0, 0.5.0, 0.6.0, 0.7.0, 0.8.0, 0.9.0` reported when I try to install 1.9.0
the rest of packages have the same issue as the two above, so I had to either upgrade or to point to a similar version

Reply:
Interestingly we've been using the wrong python package for a while: it's the [python-saml](https://github.com/mehcode/python-saml), not the saml one.